tools/prom-metrics-collector/metrics_collector.go

```diff
@@ -68,11 +59,8 @@ func ReadMetrics() []*dto.MetricFamily {
 		}
 	}
 	for _, cdiMetric := range metricsList {
-		// Remove ignored metrics from all rules
-		if _, isExcludedMetric := excludedMetrics[cdiMetric.Name]; !isExcludedMetric {
-			mf := parser.CreateMetricFamily(cdiMetric)
-			metricFamily = append(metricFamily, mf)
-		}
+		mf := parser.CreateMetricFamily(cdiMetric)
```

Comment:
Why did you remove this? Please keep the code for future use if needed.
Just remove the metrics from 
excludedMetrics

Reply:
+1 Done